index.html

```diff
@@ -296,15 +296,18 @@ <h2 class="heading-2">Contact me</h2>
                             <textarea placeholder="Write your message here" id="message" name="message" maxlength="500"
                                 required></textarea>
```

Comment:
Great design team 🤝

- [ ] [Optional] You may kindly make your text area similar to the design provided in the Figma template.
- Your design
![form](https://user-images.githubusercontent.com/63917725/180623238-eca0ba3a-c419-4fe8-84ee-03c5174ff210.PNG)

- Figma design
![Figma](https://user-images.githubusercontent.com/63917725/180623291-9a63ba11-fe6b-43ba-9a86-1924ae891356.PNG)